plutarch.cabal

```diff
@@ -169,12 +169,14 @@ test-suite examples
     , base
     , bytestring
     , generics-sop
+    , hedgehog
     , plutarch
     , plutus-core
     , plutus-ledger-api
     , plutus-tx
     , rank2classes       >=1.4.4
     , tasty
+    , tasty-hedgehog
```

Comment:
We've committed to using sydtest rather than tasty.

Reply:
sorry about that, I forgot to remove the import when moving to sydtest.